qemu/qemu.go

```diff
@@ -2300,6 +2300,9 @@ type Config struct {
 	// Bios is the -bios parameter
 	Bios string
 
+	// PFlash is the -pflash parameter
```

Comment:
I'd reword this comment: `PFlash specifies the parallel flash images (-pflash parameter)`

Reply:
thanks @devimc